bin/github_push_artifacts.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# PREREQUISITES
+#
+# ghr
+
+TAG=$1
+ARTIFACTS=$2
+GITGUB_TOKEN=$3
+
+ghr -t $GITGUB_TOKEN $TAG $ARTIFACTS 
```

Comment:
I don't see any benefit with having a shell script here. Use shell scripts when there is complicated bash logic/flow control.  Use inline commands in make targets when there's just a one-liner.